style.css

```diff
@@ -108,8 +108,21 @@ body {
   height: auto;
   opacity: 0.6;
 }
+/****************************       Salmon Events  *************************/
```

Comment:
General note: Lots of redundant CSS rules here. You can abstract some of these rules that are reused in several places and put them into class selectors. Then apply the classes to elements that need those rules. 

I would also like you to work with more pseudo-selectors, rather than getting deep with ID's and other generic selectors.